command/base_helpers.go

```diff
@@ -238,6 +238,14 @@ func humanDuration(d time.Duration) string {
 
 // humanDurationInt prints the given int as if it were a time.Duration  number
 // of seconds.
-func humanDurationInt(i int) string {
-	return humanDuration(time.Duration(i) * time.Second)
+func humanDurationInt(i interface{}) interface{} {
+	switch i.(type) {
+	case int:
+		return humanDuration(time.Duration(i.(int)) * time.Second)
+	case int64:
```

Comment:
I think we should add a case for `json.Number` here as well (like I did in the PR), since it's common that we are parsing a response that might have been JSON. If that errors, just return the original `i`.

Reply:
I did you one better, got rid of the duplication entirely.